bberg/src/circuit_builder.rs

```diff
@@ -86,12 +87,12 @@ impl CircuitBuilder for BBFiles {
                     relation_name = relation_name
                 )
         };
-        let check_permutation_transformation = |permutation_name: &String| {
+        let check_lookup_transformation = |lookup_name: &String| {
             format!(
-                    "if (!evaluate_permutation.template operator()<honk::sumcheck::{permutation_name}_relation<FF>>(\"{permutation_name}\")) {{
+                    "if (!evaluate_logderivative.template operator()<honk::sumcheck::{lookup_name}_relation<FF>>(\"{lookup_name}\")) {{
                         return false;
                     }}",
-                    permutation_name = permutation_name
+                    lookup_name = lookup_name
```

Comment:
nitpick: I would prefer two different identifiers, unless this is Rust standard practice.